src/dotnet/Fable.JS/bench/app.fs

```diff
@@ -58,12 +64,17 @@ let main argv =
         let parseFSharp () = fable.ParseFSharpProject(checker, fileName, source)
         let parseFable ast = fable.CompileToBabelAst(fableCoreDir, ast, fileName, optimized)
         let bench i =
-            let ms1, fcsAst = measureTime parseFSharp ()
-            fcsAst.Errors |> Array.iter (printfn "Error: %A")
-            if (Array.length fcsAst.Errors) > 0 then failwith "Too many errors."
-            let ms2, babelAst = measureTime parseFable fcsAst
+            let ms1, fsAst = measureTime parseFSharp ()
+            let errors = fable.GetParseErrors fsAst
+            errors |> Array.iter (printfn "Error: %A")
+            if errors.Length > 0 then failwith "Too many errors."
+            let ms2, babelAst = measureTime parseFable fsAst
+            if i = 1 then
+                // fable.PrintFSharpAst (fsAst, false, printfn "%s")
+                // printfn "Babel AST: %s" (toJson babelAst)
+                // writeAllText fsAstFile (toJson fsAst)
+                writeAllText babelAstFile (toJson babelAst)
```

Comment:
Is `toJson` working? JSON de/serialization has been moved out of Fable.Core for Fable 2, and `toJson` is not defaulting to `JSON.stringify` but I don't think this will work with types in Babel AST.

Reply:
I am not sure it's a good idea to let `toJson` in Fable 2.

Because, people could think it's still working as before when in fact it isn't.

Reply:
@MangelMaxime I had to leave `toJson` to make Fable 2 alpha work because some libs still use (for example, Elmish to print values to console). It's marked as obsolete, but hopefully the libs will be updated and we can remove it for the stable release.

@ncave I've mode [some changes in Babel AST types to prevent mangling of field names](https://github.com/fable-compiler/Fable/commit/f672dc24aa365724d4c8a65156e8228857e23275) and other (use arrays instead of list, immediately convert unions into strings...). In theory, it should be possible to send those types to Babel now without any JSON serialization at all.